src/renderer/features/balances/components/AmountInput.tsx

```diff
@@ -196,3 +197,71 @@ export const AmountInput = ({
     />
   );
 };
+
+const DEFAULT_LEFT_PADDING = 11;
+const EXTENDED_LEFT_PADDING = 19;
+
+type InputProps = {
+  name?: string;
+  value: string;
+  placeholder: string;
+  label: ReactNode;
+  invalid?: boolean;
+  disabled?: boolean;
+  prefixElement: ReactNode;
+  suffixElement?: ReactNode;
+  onChange: (value: string) => void;
+};
+export const Input = ({ label, invalid, disabled, prefixElement, suffixElement, onChange, ...props }: InputProps) => {
```

Comment:
Maybe split it for two files?

Reply:
Left 2 components in a single file, `AmountInput` requires refactoring anyway, did't want to do much while updating  `Inputs`